kolibri/core/content/utils/channel_import.py

```diff
@@ -952,7 +953,6 @@ def import_channel_data(self):
 
     def run_and_annotate(self):
         try:
-            self.current_channel = ChannelMetadata.objects.get(id=self.channel_id)
             old_order = self.current_channel.order
```

Comment:
Having moved the `ChannelMetadata` lookup to the constructor, looking up `old_order` no longer requires the `try/except`, but it should instead check if `self.current_channel` is defined and then set it otherwise have it be `None`.